assets/scss/components/banner-subservices.scss

```
@@ -0,0 +1,72 @@
+.banner-subservices {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  margin-bottom: 493px;
```

On peut arrondir ici 
```suggestion
  margin-bottom: 490px;
```